coriolisclient/v1/migrations.py

```diff
@@ -47,8 +47,10 @@ def get(self, migration):
         return self._get('/migrations/%s' % base.getid(migration), 'migration')
 
     def create(self, origin_endpoint_id, destination_endpoint_id,
-               destination_environment, instances, network_map,
+               destination_environment, instances, network_map=None,
                skip_os_morphing=False):
+        if not network_map:
+            network_map = destination_environment['network_map']
```

Comment:
This introduces a whole other `KeyError` opportunity; so please just `destination_environment.get('network_map', {})`